hera_pspec/pspecdata.py

```diff
@@ -1526,8 +1526,9 @@ def get_Q(self, mode, feed_pol=False):
         mode : int
             Central wavenumber (index) of the bandpower, p_alpha.
 
-        feed_pol : str/int/bool, optional
-            Polarization for the beam. If False, isotropic beam would be returned.
+        pol : str/int/bool, optional
+            Polarization for the beam. In case the polarization is not found, 
+            isotropic beam would be returned.
```

Comment:
Still not quite sure what this means. Does "isotropic beam" = Stokes I or similar?

Reply:
It's a uniform beam (unity response at all angles and frequencies). In this case, all beam integrals in theta will be 4*pi. This would only be used (with a warning) when `exact_norm=True` and no beam is supplied.

Reply:
Thanks for this. I've updated the docstring appropriately. Ready to merge!